subsys/logging/Kconfig

```diff
@@ -249,6 +249,23 @@ config LOG_BACKEND_UART_FORMAT_TIMESTAMP
 	default y
 	help
 	  When enabled timestamp is formatted to hh:mm:ss:ms,us.
+	  
+config LOG_FRONTEND
+	bool "Enable GPIO frontend"
+	help
+	  When enabled frontend is using GPIO pins to output logs.
+
+config LOG_FRONTEND_PORT_NUMBER
+	int "Choose port number"
+	depends on LOG_FRONTEND
+	default 1
```

Comment:
note that nrf51 and nrf52832/nrf52810 does not have 2 gpio ports. In that case default 1 is wrong. Check out arch/arm/soc/nordic_nrf/kconfig.peripherals. You can use configs from there. e.g *default 1 if HAS_HW_NRF_GPIO1*